src/components/activity-header/profile-nav-header.scss

```diff
@@ -4,19 +4,17 @@
   display: flex;
   justify-content: flex-end;
   width: $content-width;
-  color: white;
-  margin-top: 5px;
+  align-items: center;
 
   &.full {
     width: 100%;
   }
 
   .profile-name {
-    height: 20px;
-    padding: 0 5px 0 5px;
-    background-color: #414546;
-    font-size: 13px;
-    line-height: 20px;
+    padding: 7px;
+    font-size: 16px;
+    background-color: #8282824e;
+    color: var(--theme-primary-text-color);
```

Comment:
We'll need to be a little careful about setting this to the primary text color since the text color will sometimes be light and sometimes be dark.  Since the background is static, this could cause readability issues in some cases.

Reply:
I still think there is a visibility issue here.  Look at what we get with the Waters theme colors:
![image](https://user-images.githubusercontent.com/5126913/89845379-28a5f000-db33-11ea-887e-8f574622680c.png)
I would vote that this font color should be static and the background (background-color in the class profile-name) should be darker so it is always legible.  What do you think?

Reply:
After a little tinkering, I vote that the CSS class `.profile-name` should have the following styles:
`color: $light-font-color;`
`background-color: #414546;`

I don't think we can have the semi-transparent background in this case because there is far too much variability in what appears behind it.  I think the best bet is to make the background and color static and remove the transparency.  What do you think?

Reply:
How does this look?
![colors](https://user-images.githubusercontent.com/5126913/89864297-91f02800-db60-11ea-9b4b-bb1c92682148.gif)